src/test/java/pl/gacik/coordinates/SimpleCoordinates2DTests.java

```diff
@@ -0,0 +1,158 @@
+package pl.gacik.coordinates;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.testng.Assert;
+
+import java.util.Random;
+
+@Test(groups = {"SimpleCoordinates2DTests"})
+public class SimpleCoordinates2DTests {
+
+    @DataProvider(name = "10randomCoordinates")
+    public static Object[][] randomCoordinates() {
```

Comment:
Even if it's a test, data shouldn't be random. Testing data also should be predicted.